fdbserver/TenantBalancer.actor.cpp

```diff
@@ -776,15 +870,23 @@ ACTOR Future<Void> cleanupMovementSource(TenantBalancer* self, CleanupMovementSo
 	++self->cleanupMovementSourceRequests;
 
 	try {
-		// TODO once the range has been unlocked, it will no longer be legal to run cleanup
 		state std::string tenantName = req.tenantName;
+		// TODO once the range has been unlocked, it will no longer be legal to run cleanup
+		TraceEvent("TenantBalancerClearErase").detail("TenantName", tenantName);
```

Comment:
In addition to the below, we should also add the cleanup type here.

```suggestion
		TraceEvent(SevDebug, "TenantBalancerClearErase", self->tbi.id()).detail("TenantName", tenantName);
```